hive/configs/marlgrid/rainbow.yml

```diff
@@ -106,6 +107,7 @@ agents:
           steps: 10000
       min_replay_history: 500
       batch_size: 32
+      device: 'cpu'
```

Comment:
Why was this added?

Reply:
I didn't notice that now the agent itself recognizes the device. Will remove it.

Reply:
Actually, since marlgrid experiments should be run on gpu anyway, I changed it to cuda.